Artroom1/HomeViewController.swift

```diff
@@ -39,12 +41,12 @@ import UIKit
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        assignbackground()
-        
+        assignbackgrounds()
+//        assignTheBlur()
```

Comment:
![I see what you did there ...](https://media3.giphy.com/media/rApKuVWCZZgvm/giphy.gif?cid=348844935bea94dc6957485641c51310)